build/configs/rtl8721csm/scripts/rlx8721d_img2_ns.ld

```diff
@@ -109,11 +109,90 @@ SECTIONS
 
 	.ram_image2.text :
 	{
+		__ram_text_start__ = .;
 		_stext = ABSOLUTE(.);
 		*(.image2.ram.text*)
 		*(.image2.net.ram.text*)
+		/*os*/
```

Comment:
@sangam-swami @kishore-sn @jeongchanKim @jeongarmy Could you check this? Is this ok to do binary recovery?